src/main/java/gift/MemberController.java

```diff
@@ -28,11 +29,10 @@ public ResponseEntity<String> register(@RequestBody @Valid Member member) {
     @PostMapping("/login")
     public ResponseEntity<String> login(@RequestBody @Valid Member member) {
         String token = memberService.login(member);
-        if(token =="NoEmail"){
-            return ResponseEntity.ok().body("{\"error\": \"존재하지 않는 이메일입니다.\"}");
-        }
-        if(token == "inValidPassword"){
-            return ResponseEntity.ok().body("{\"error\": \"잘못된 비밀번호 입니다.\"}");
+        if(token.equals("NoEmail")){
```

Comment:
```suggestion
        if ("NoEmail".equals(token)) {
```
공백...! 전체 프로젝트에 대해 한번씩 다 확인해주시면 좋을 것 같아요.
그리고 if 문 안의 로직을 위와 같이 바꿔보면 어떤 점이 다를까요? 꽤 중요한 내용이니 한 번 고민해보아요.

Reply:
token.equals("NoEmail")를 사용할 경우 token이 null일 때 NullPointerException이 발생 할 수 있고, 
if ("NoEmail".equals(token))을 사용할 경우 token이 null이면 false가 반환 됩니다.
null 안정성을 위해서 수정하는게 좋을 것 같습니다. 감사합니다!!